server/database/migrations/20190528150355_shippingAddresses.js

```diff
@@ -0,0 +1,22 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('shippingAddresses', (table) => {
+    table.increments();
+    table.string('street', 100);
+    table.string('aptSuite', 100);
+    table.string('city', 100);
+    table.string('state', 100);
```

Comment:
Should we decrease to 50 characters?  No state has this many characters... Just to save space in DB.  Idk how this works actually haha